PowerEditor/src/NppBigSwitch.cpp

```diff
@@ -2106,6 +2106,18 @@ LRESULT Notepad_plus::process(HWND hwnd, UINT message, WPARAM wParam, LPARAM lPa
 			return TRUE;
 		}
 
+		case NPPM_INTERNAL_WINSESSIONEXIT:

```

Comment:
Add 4 more characters DOWS won't make it much longer but make it much more clear :)